Dockerfile

```diff
@@ -75,6 +75,10 @@ RUN pip install --no-cache-dir pre-commit==2.17.0
 COPY .pre-commit-config.yaml /code/timemachine/
 RUN cd /code/timemachine && git init . && pre-commit install-hooks
 
+# Install CI requirements
+COPY ci/requirements.txt /code/timemachine/ci/requirements.txt
```

Comment:
If we are going to expand this dockerfile to support more CI logic, we should create a separate layer just for CI. Otherwise right now these ci requirements will get baked into production containers

Reply:
Should be relatively straight forward, just replicate the 'final' docker stage and run the two CI specific commands within it and change the docker target in CI.

Seems best to do it in this PR, unless it isn't as trivial as I am imagining

Reply:
Good call, done in 5d36eb4 (and fixed ordering in ba2d125)